sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkTracerProviderBuilder.java

```diff
@@ -94,14 +98,53 @@ public SdkTracerProviderBuilder addSpanProcessor(SpanProcessor spanProcessor) {
     return this;
   }
 
+  /**
+   * Adds a {@link SpanExporter} to the end of the span pipeline, enabling default batching of spans
+   * before export. To configure batching or disable it, use {@link #addExporter(SpanExporter,
+   * BatchSettings)} instead.
+   */
+  public SdkTracerProviderBuilder addExporter(SpanExporter exporter) {
+    requireNonNull(exporter, "exporter");
+    return addExporter(exporter, BatchSettings.builder().build());
+  }
+
+  /**
+   * Adds a {@link SpanExporter} to the end of the span pipeline. {@link BatchSettings} will
+   * configure the batching used before exporting spans. If {@link BatchSettings#noBatching()} is
+   * provided, no batching will be done.
+   */
+  // Using deprecated for public use class
+  @SuppressWarnings({"deprecation", "UnnecessarilyFullyQualified"})
+  public SdkTracerProviderBuilder addExporter(SpanExporter exporter, BatchSettings batchSettings) {
+    requireNonNull(exporter, "exporter");
+    requireNonNull(batchSettings, "batchSettings");
+    if (batchSettings == BatchSettings.noBatching()) {
+      exporterSpanProcessors.add(
+          io.opentelemetry.sdk.trace.export.SimpleSpanProcessor.create(exporter));
```

Comment:
odd fully qualified import here

Reply:
It seems to be the only way to avoid the `deprecation` warning that would other get applied to the import statement itself amazingly.

Reply:
weird!